src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -156,7 +167,8 @@
    request :- Request
    name :- schema/Str]
   (if-let [ { matched-rule :rule matches :matches } (some #(match? % request) rules)]
-    (if (acl/allowed? (:acl matched-rule) name (:remote-addr request) matches)
+    (if (and (true? (get-in request is-authentic-key))
```

Comment:
@nwolfe This is where I was thinking we'll want to implement [TK-260](https://tickets.puppetlabs.com/browse/TK-260).  Until then, all unauthenticated requests are denied.